scss/boost_union/post.scss

```diff
@@ -917,3 +914,994 @@ body.pagelayout-login:not(.loginbackgroundimage) #footnote {
+    + .no-overflow table tbody tr td ul li {
+        label {
+            margin-right: 10px;
+        }
+    }
+}
+
+/* The Menu settings - Title alignment in the table block */
+.main-inner-wrapper .main-inner div[role="main"] {
+    .no-overflow .generaltable tbody tr td {
+        /*  In the Meu settings page, the Smart menu text in the table as display as inline-block
+            so the help icon stays near the Smart menu text  */
+        .menu-title {
+            display: inline-block;
+        }
+        /* Margin left added for the help icon near the Smart menu text in the table */
+        .btn-link {
+            margin-left: 5px;
+        }
+    }
+    /* Added margin bottom for the "Create new Smart menu" button in the Menu settings page */
+    > .singlebutton {
+        margin-bottom: 20px;
+    }
+}
+/* End of Smart Menu settings */
+
+/* Footer Menu */
+#page-wrapper #page-footer {
+    .navbar {
+        /* Footer bottom menu */
+        /* Padding space removed around the bottom menu block, box-shadow and z-index
+        to view the bottom menu at the top added*/
+        &.boost-union-bottom-menu {
+            padding: 0;
+            box-shadow: 0px 0px 6px 0 rgba(0, 0, 0, 0.1);
+            z-index: 1050;
+            .bottom-navigation .more-nav {
+                > li {
+                    /* The footer bottom dropdown menu icon and text are displayed in separate rows. */
+                    &:not(.dropdown):not([data-key=""]) a {
+                        flex-direction: column;
+                        /* The footer bottom dropdown menu icon margin is removed. */
+                        i {
+                            margin: 0 0 3px;
+                        }
+                    }
+                    /* The footer bottom menu item border bottom is removed when it's not active. */
+                    a:not(.active) {
+                        border-bottom: 0;
+                    }
+                }
+                li {
+                    &:first-child,
+                    &:nth-child(2),
+                    &:nth-child(3) {
+                        &.dropdown .dropdown-menu .dropdown-submenu ul {
+                            left: 100%;
+                            right: auto;
+                        }
+                    }
+                    &.dropdown {
+                        /* Footer bottom dropdown menu width, height and removed margin, position to stay the dropup menu */
+                        .dropdown-menu {
+                            width: auto;
+                            margin-top: 0;
+                            margin-bottom: 0;
+                            top: auto;
+                            bottom: 100%;
+                            left: auto;
+                            right: 0;
+                            /* The footer bottom dropdown submenu min-width, max-height, overflow and position to show the menus above the footer */
```

Comment:
Do break the row to multiple lines and make sure you do not extend 132 characters of line length.

Reply:
Thanks, Luca.
I fixed this in a review commit now.